backend/internal/db/dbgorm/dbgorm.go

```diff
@@ -99,7 +99,7 @@ func (db *gormdb) Teardown(ctx context.Context) error {
 	return nil
 }
 
-// Todo: not sure this will work with the connect method
+// TODO: not sure this will work with the connect method
```

Comment:
?

Reply:
most of the tools and common practice is grep/search the code for common keywords/labels stated as CAP letters

Reply:
I though you added the comment

Reply:
resolving then